libres/old_tests/enkf/test_row_scaling.cpp

```diff
@@ -28,22 +28,18 @@
 #include <ert/util/rng.hpp>
 
 void test_create() {
-    row_scaling_type *row_scaling = row_scaling_alloc();
-    test_assert_int_equal(row_scaling_get_size(row_scaling), 0);
+    RowScaling row_scaling;
+    test_assert_int_equal(row_scaling.size(), 0);
 
-    test_assert_throw(row_scaling_iget(row_scaling, -1), std::out_of_range);
-    test_assert_throw(row_scaling_iget(row_scaling, 1000), std::out_of_range);
+    test_assert_throw(row_scaling[-1], std::out_of_range);
+    test_assert_throw(row_scaling[1000], std::out_of_range);
 
-    test_assert_throw(row_scaling_iset(row_scaling, -1, 0), std::out_of_range);
-    test_assert_throw(row_scaling_iset(row_scaling, 0, -1),
-                      std::invalid_argument);
-    test_assert_throw(row_scaling_iset(row_scaling, 0, 2),
-                      std::invalid_argument);
+    test_assert_throw(row_scaling.assign(-1, 0), std::out_of_range);
+    test_assert_throw(row_scaling.assign(0, -1), std::invalid_argument);
+    test_assert_throw(row_scaling.assign(0, 2), std::invalid_argument);
 
-    row_scaling_iset(row_scaling, 9, 0.25);
-    test_assert_double_equal(row_scaling_iget(row_scaling, 9), 0.25);
-
-    row_scaling_free(row_scaling);
+    row_scaling.assign(9, 0.25);
+    test_assert_double_equal(row_scaling[9], 0.25);
 }
 
 static void scaleX(matrix_type *X, const matrix_type *X0, double alpha) {
```

Comment:
Any reason to re-implement `scaleX` in the test?
It is already implemented in `row_scaling.cpp`.

Reply:
Nope, but it's outside the scope of this PR